src/main/python/ayab/ayab.py

```diff
@@ -549,14 +358,19 @@ def getSerialPorts(self):
         """
         return list(serial.tools.list_ports.grep("USB"))
 
-    def slotPlaysound(self, event):
-        return
-        # if event == "start":
-        #     playsound(self.app_context.get_resource("assets/start.wav"))
-        # if event == "nextline":
-        #     playsound(self.app_context.get_resource("assets/nextline.wav"))
-        # if event == "finished":
-        #     playsound(self.app_context.get_resource("assets/finish.wav"))
+    def playsound(self, sound):
+        if str2bool(self.prefs.settings.value("quiet_mode")):
+            return 
+        dirname = path.join("..", "..", "resources", "base", "assets", path.dirname(__file__))
```

Comment:
When running the application, the .wav files are not found. (`WARNING  File start.wav not found.`)
Please use self.app_context.get_resource() (see before your commit) to get an OS independent location of the files.